repos/tap/theme/components/definitions/definitionList.js

```diff
@@ -88,21 +88,25 @@ const listItem = theme => {
         main: {
           ...theme.transition([ 'opacity' ], 0.8),
           opacity: 0,
+          position: 'absolute',
+          right: '0',
+          pR: theme.padding.size,
         },
         action: {
           main: {
           },
           touchable: {
-            flD: 'row',
+            flD: 'column',
           },
           icon: {
             icon: {
-              c: tapColors.inactive,
+              c: tapColors?.primary,
               mR: 5,
+              paddingLeft: 5
             },
           },
           name: {
-            ftSz: 12,
+            ftSz: 9,
             ftWt: 'bold',
             c: tapColors.inactive,
```

Comment:
Because of the update to line 100, you should also update this line to be: `c: tapColors?.primary,`
That way they are consistent

Reply:
Done . Along with that , I also removed the name element in hover since , if all we are doing is changing opacity from 0 to 1 , we don't need to specify the color in 2 places